src/components/language-select/language-toggle/template.njk

```diff
@@ -0,0 +1,15 @@
+<nav class="language-select {{params.class}}" aria-label="{{params.ariaLabel}}">
+    <ul class="govuk-list language-select__list">
+        {% for pageIndex,lang in params.languages.entries() %}
+            <li class="language-select__list-item" tabindex="{{pageIndex}}">
```

Comment:
Is setting the tabindex here required? The span tag won't be focusable as it's the current language and `a` tags are focusable by default

Reply:
Put tabIndex at the li tag level seems to be a good practice when it's about accessibility.
(https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets)
But tell me if you want to remove it or improve it.

Reply:
What values are we expecting for pageIndex?

Reply:
pageIndex comes from the for loop ({% for pageIndex,lang in params.languages.entries() %}).
It's just the index of the loop.

Reply:
I removed tabindex for li tags